faq.rst

```diff
@@ -109,23 +109,25 @@ How can I secure the access to my web application?
 OpenShift supports secure routes and everything is prepared on APPUiO to have
 it secured easily. Just edit the route and change the termination type to ``edge``.
 There is a default trusted certificate in place for ``*.appuioapp.ch`` which is
-used in this case. If you want to use your own certificate, see `Routes <https://docs.openshift.com/enterprise/latest/dev_guide/routes.html>`__.
+used in this case. If you want to use your own certificate, see `Routes
+<https://docs.openshift.com/enterprise/latest/dev_guide/routes.html>`__.
 
 Can I run a database on APPUiO?
 -------------------------------
 
-We provide shared persistent storage using GlusterFS. Please make sure that the database intended to use is capable
-of storing it's data on a shared filesystem. We don't recommend running production databases with GlusterFS as
-storage backend. If you don't fear it, we strongly urge you to at least have a regular database dump available.
-On ``https://github.com/appuio?q=backup`` you can find some database dump applications.
-For highly-available and high-performance managed databases, please contact the APPUiO team under `hello`_.
+Short answer: Yes. But you have to be vigilant. Use the ``gluster-database``
+storage class as described in :ref:`persistent-storage`. See
```

Comment:
**Side note:** We don't actually provide a code sample on how to set the storage class, anywhere in the docs. This is also hard to find on the web.

Reply:
how is it hard to find on the web? It's documented quite thoroughly in the official K8s docs: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1

Reply:
Your link points to a paragraph of text that needs to be read, understood and converted into code. Whether we like it or not, we should give StackOverflow:tm: style answers. Code snippets that "just work", or links to such code snippets.

The K8s documentation on [StatefulSets](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#components) contains such a snippet. But you have to know that you need to look out for `storageClassName`. I see that also the page you link to [provides a similarly useful snippet](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistent-volumes). I'll try to include pointers to that in the APPUiO docs. Thanks! :+1: